sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -425,7 +425,9 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
 
         logWarning(s"CREATE TEMPORARY TABLE ... USING ... is deprecated, please use " +
```

Comment:
@cloud-fan Thanks for reviewing!  I found that this deprecation was added by your PR #14482 back on 8/5/2016.  Do you think it is the right time to remove `CREATE TEMPORARY TABLE...` completely?